W-17038766 payload storage updates (#245)
mulesoft/docs-partner-manager: modules/ROOT/pages/setup-payload-storage-API.adoc

To use your own storage solution, you must implement an API that provides Partner Manager with a standardized way to connect to your storage.

> NOTE: You must make the payload storage API publicly accessible if you want the payload to be available from the Partner Manager Activity page. If your payload storage API can only be accessed in your network, then you must use a different implementation of the API. See <<payload-view, Payload view URL>> for more information.

Reviewer: ```suggestion
NOTE: If you want the payload to be available from the Partner Manager Activity page, the payload storage API must be publicly accessible. If your payload storage API is accessible only through your network, you must implement the API differently. See <<payload-view, Payload view URL>> for more information.```
